test/integration/testdata/defaults/default.go

```diff
@@ -15,8 +15,23 @@ type Defaulted struct {
 
 	// +default={"foo": "bar"}
 	Map map[string]Item
+
+	// +default=ref(ConstantValue)
+	LocalSymbolReference string `json:"localSymbolReference,omitempty"`
+	// +default=ref(k8s.io/kube-openapi/test/integration/testdata/defaults.ConstantValue)
+	FullyQualifiedSymbolReference string `json:"fullyQualifiedSymbolReference,omitempty"`
+	// +default=ref(k8s.io/kube-openapi/test/integration/testdata/enumtype.FruitApple)
+	ExternalSymbolReference string `json:"externalSymbolReference,omitempty"`
+	// +default=ref(k8s.io/kube-openapi/test/integration/testdata/enumtype.FruitApple)
+	PointerConversionSymbolReference *DefaultedItem `json:"pointerConversionSymbolReference,omitempty"`
+	DefaultedAliasSymbolReference    DefaultedItem  `json:"defaultedAliasSymbolReference,omitempty"`
```

Comment:
So here we would end-up with what if you had the same comment as the line above. IOW, 
```suggestion
         // +default=ref(k8s.io/kube-openapi/test/integration/testdata/enumtype.FruitApple)
	DefaultedAliasSymbolReference    DefaultedItem  `json:"defaultedAliasSymbolReference,omitempty"`
```
would give "apple" or "SymbolConstant"? Because I find it hard to predict for people who will author these types to predict if it doesn't have the same behavior as above.

Reply:
What you wrote would give `apple`. The principle is simple and mirrors the same behavior in defaulter-gen: the default defined closest to your field is going to take precedence.

> Because I find it hard to predict for people who will author these types to predict if it doesn't have the same behavior as above.

I don't understand this

Reply:
If it gives apple, then we're fine. I thought there was a difference between pointer/non-pointer or something subtle hidden somewhere.